src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_checkout.yml

```
@@ -37,29 +37,34 @@ winzou_state_machine:
                     on: ["select_shipping", "address", "select_payment", "skip_shipping", "skip_payment"]
                     do: ["@sylius.order_processing.order_processor", "process"]
                     args: ["object"]
+                    priority: 0
                 sylius_create_order:
                     on: ["complete"]
                     do: ["@sm.callback.cascade_transition", "apply"]
                     args: ["object", "event", "'create'", "'sylius_order'"]
+                    priority: 0
                 sylius_save_checkout_completion_date:
                     on: ["complete"]
                     do: ["object", "completeCheckout"]
                     args: ["object"]
+                    priority: 100
                 sylius_skip_shipping:
                     on: ["address"]
                     do: ["@sylius.state_resolver.order_checkout", "resolve"]
                     args: ["object"]
-                    priority: 1
+                    priority: 100
                 sylius_skip_payment:
                     on: ["select_shipping"]
                     do: ["@sylius.state_resolver.order_checkout", "resolve"]
                     args: ["object"]
-                    priority: 1
+                    priority: 100
                 sylius_control_payment_state:
                     on: ["complete"]
                     do: ["@sylius.state_resolver.order_payment", "resolve"]
                     args: ["object"]
+                    priority: 0
```

As there are priorities for `complete` transition already defined:
```suggestion
                    priority: 200
```